kernel/posix/pthread_sched.c

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright (c) 2018 Intel Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <kernel.h>
+#include <posix_sched.h>
+
+int sched_get_priority_min(int policy)
+{
+	if (!((policy == SCHED_FIFO) || (policy == SCHED_RR))) {
```

Comment:
You can apply [De Morgan's Law](https://en.wikipedia.org/wiki/De_Morgan%27s_laws) here and simplify this expression.

Reply:
Sure I will change it to  - if((policy != SCHED_FIFO) && (policy != SCHED_RR)))

Reply:
No need for the extra parenthesis.